CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.1.0]
+### Changed
```

Comment:
This is a bug fix, so it should be v14.0.1 and categories as "Fixed"

Reply:
PR is updated.